src/router.js

```diff
@@ -0,0 +1,23 @@
+import { createWebHistory, createRouter } from 'vue-router'
+import HelloWorld from './components/HelloWorld.vue'
+
+const routes = [
+  {
+    path: '/',
+    name: 'Home',
+    component: HelloWorld,
+  },
+  //Can add this in once search results is merged
```

Comment:
Don't preempt adding things. Add them when they are needed. Delete this commented code.